dvc/commands/data.py

```diff
@@ -0,0 +1,178 @@
+import argparse
+import logging
+from typing import TYPE_CHECKING
+
+from funcy import compact, log_durations
+
+from dvc.cli.command import CmdBase
+from dvc.cli.utils import append_doc_link, fix_subparsers
+from dvc.ui import ui
+
+if TYPE_CHECKING:
+    from dvc.repo.data import Status as DataStatus
+
+
+logger = logging.getLogger(__name__)
+
+
+class CmdDataStatus(CmdBase):
+    COLORS = {
+        "not_in_cache": "red",
+        "committed": "green",
+        "uncommitted": "yellow",
+        "untracked": "cyan",
+    }
+    LABELS = {
+        "not_in_cache": "Not in cache",
+        "committed": "DVC committed changes",
+        "uncommitted": "DVC uncommitted changes",
+        "untracked": "Untracked files",
+        "unchanged": "DVC unchanged files",
+    }
+    HINTS = {
+        "not_in_cache": 'use "dvc pull <file>..." '
```

Comment:
I thought we used back ticks in these texts? E.g.
```suggestion
        "not_in_cache": 'Use `dvc pull <file>` '
```